examples/zone-printer/README.md

```diff
@@ -1,258 +1,252 @@
-you can create a project by following the instructions at
-[Creating and Managing Projects](https://cloud.google.com/resource-manager/docs/creating-managing-projects).
+    Ensure that you have a GCP Project. If you don't have one already,
+    you can create a project by following the instructions at
+    [Creating and Managing Projects](https://cloud.google.com/resource-manager/docs/creating-managing-projects).
 
-After your project is created, run the following command in your
-terminal:
+    After your project is created, run the following commands in your
+    terminal:
 
-```shell
-PROJECT=<your-project-name>
-```
+    ```shell
+    PROJECT=<your-project-name>
+    ```
 
-### 0.2 `gcloud` installed and initialized
+2. **Set up `gcloud`**:
 
-You can install `gcloud` by following the instructions at
-[Installing Cloud SDK](https://cloud.google.com/sdk/downloads) and
-initialize it by running the following commands:
+    You can install `gcloud` by following the instructions at
+    [Installing Cloud SDK](https://cloud.google.com/sdk/downloads) and
+    initialize it by running the following commands:
 
-```shell
-gcloud init # Set $PROJECT as the default project
-gcloud auth login
-gcloud auth application-default login
-```
+    ```shell
+    gcloud init # Set $PROJECT as the default project
+    gcloud auth login
+    gcloud auth application-default login
+    ```
 
-### 0.3 kubectl
+3. Set up **`kubectl`**:
 
-You can install `kubectl` by running:
+    You can install `kubectl` by running:
 
-```shell
-gcloud components install kubectl
-```
+    ```shell
+    gcloud components install kubectl
+    ```
 
-### 0.4 Kubernetes clusters on GCP
+## 1. Create Kubernetes Clusters
 
-#### Existing clusters
+`kubemci` requires Kubernetes clusters that are v1.8.1 or newer. (You can check
+your cluster version using the `kubectl version` command.)
 
-> **Note:**: If you don't already have Kubernetes cluster on GCP,
-skip to the next subsection on [creating new clusters](#creating-new-clusters).
+You will need at least two Kubernetes clusters in two different compute [zones]
+to verify that `kubemci` works.
 
-`kubemci` requires Kubernetes clusters that are v1.8.1 or newer. Please
-ensure that all the clusters you are using in this tutorial satisfy the
-minimum version requirement. You can check your cluster version using
-the `kubectl version` command.
+[zones]: https://cloud.google.com/compute/docs/regions-zones/
 
-If you already have clusters with their credentials in your local
-kubeconfig (`$HOME/.kube/config` by default) file, you need to ensure
-that you either only have clusters you are going to use for this
-tutorial in your default kubeconfig file or create a new kubeconfig
-file that just contains credentials for those clusters. Assuming that
-the clusters you are going to use for this tutorial are called
-`us-east`, `eu-west` and `asia-east`.
+#### Using existing GKE clusters
```

Comment:
seems like the tutorial would flow better if you swapped the order of this and the next subsections:
1 +#### Creating new GKE clusters
2 +#### Using existing GKE clusters

what do you think?

Reply:
yeah that makes sense.